tests/subsys/settings/functional/nvs/testcase.yaml

```diff
@@ -1,4 +1,4 @@
 tests:
   system.settings.functional.nvs:
-    platform_whitelist: qemu_x86
+    platform_whitelist: qemu_x86 native_posix
```

Comment:
64-bit support could be tested by adding `native_posix_64`  to the white list

Reply:
added :)